docs/src/hal/rtcomps.adoc

```diff
@@ -87,32 +92,32 @@ On the step type and control type selected.
 [[sec:stepgen-parameters]]
 === Parameters(((HAL stepgen parameters)))
 
-* '(float) stepgen.<chan>.position-scale' - Steps per position unit.
+* '(float) stepgen._<chan>_.position-scale' - Steps per position unit.
```

Comment:
Have you checked if that renders correctly?

Reply:
I have not, admittedly.

Reply:
Have now checked in vs code - is fine, would not expect anything else because of the period appearing frequently at the end of sentences and thus needing a low operator precedence.